Uplift/Controllers/HomeViewController+Extensions.swift

```diff
@@ -80,25 +79,24 @@ extension HomeViewController: UICollectionViewDelegate, UICollectionViewDelegate
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
         let width = collectionView.frame.width
 
-        switch sections[indexPath.section] {
-        case .todaysClasses:
-            let padding: CGFloat = 20.0
-            let cellWidth = gymClassInstances.isEmpty ? width - 2.0 * padding : width
-            return CGSize(width: cellWidth, height: TodaysClassesListCell.totalHeight)
-        case .yourActivities:
-            let height = ActivitiesListCell.itemHeight
-            return CGSize(width: width, height: height)
-        case .myGyms:
-            //Height is calculated as item height of every row of 2 cells + minimum line spacing between them + bottom section inset value
-
-            //Height of all cells (in rows of 2), plus line spacings after each of them
-            //MARK: myGyms gyms.count and removed / 2
-            var height: CGFloat = (GymsListCell.itemHeight + GymsListCell.minimumItemSpacing) * ceil(CGFloat(integerLiteral: gyms.count))
-
-            //Subtract extra minimum line spacing below the last row of cells, and add the section inset
+        switch sections[indexPath.section] {        case .fitnessCenters:
```

Comment:
The indentation looks a little off here, not sure if it's just Github but the switch and case are on the same line :(